Userland/Libraries/LibThreading/ThreadPool.h

```diff
@@ -22,26 +22,37 @@ struct ThreadPoolLooper {
     {
         Optional<typename Pool::Work> entry;
         while (true) {
+            pool.m_busy_count++;
```

Comment:
This should probably be a RAII style guard similar to `MutexLocker` but was dropped to making this PR do too many things

Reply:
I don't think that's necessary, the logic I see here doesn't really lend itself well to that pattern.